selinux/98selinux-microos/selinux-microos-relabel.sh

```diff
@@ -51,13 +51,13 @@ rd_microos_relabel()
         . "${ROOT_SELINUX}"/etc/selinux/config
         # Marker when we had relabelled the filesystem. This is relabelled as well.
         > "${ROOT_SELINUX}"/etc/selinux/.relabelled
-        LANG=C chroot "$ROOT_SELINUX" /sbin/setfiles $FORCE -e /var/lib/overlay -e /proc -e /sys -e /dev -e /etc "/etc/selinux/${SELINUXTYPE}/contexts/files/file_contexts" $(chroot "$ROOT_SELINUX" cut -d" " -f2 /proc/mounts)
+        LANG=C chroot "$ROOT_SELINUX" /sbin/setfiles $FORCE -T 0 -e /var/lib/overlay -e /proc -e /sys -e /dev -e /etc "/etc/selinux/${SELINUXTYPE}/contexts/files/file_contexts" $(chroot "$ROOT_SELINUX" cut -d" " -f2 /proc/mounts)
         # On overlayfs, st_dev isn't consistent so setfiles thinks it's a different mountpoint, ignoring it.
         # st_dev changes also on copy-up triggered by setfiles itself, so the only way to relabel properly
         # is to list every file explicitly.
         # That's not all: There's a kernel bug that security.selinux of parent directories is lost on copy-up (bsc#1210690).
         # Work around that by visiting children first and only then the parent directories.
-        LANG=C chroot "$ROOT_SELINUX" find /etc -depth -exec /sbin/setfiles $FORCE "/etc/selinux/${SELINUXTYPE}/contexts/files/file_contexts" \{\} +
+        LANG=C chroot "$ROOT_SELINUX" find /etc -depth -exec /sbin/setfiles $FORCE -T 0 "/etc/selinux/${SELINUXTYPE}/contexts/files/file_contexts" \{\} +
```

Comment:
Actually, `-T0` might break the workaround here. The label for the parent *must* be applied after the child. If threading changes the order, it'll break.